frontend/src/app/Settings/Users/UserFacilitiesSettingsForm.tsx

```diff
@@ -127,127 +121,72 @@ const UserFacilitiesSettingsForm: React.FC<Props> = ({
     [userFacilities]
   );
 
-  const facilitiesToAdd = allFacilities
-    .filter(({ id }) => !userFacilityLookup.has(id))
-    .sort(alphabeticalFacilitySort);
-
-  const removeButtonClasses = classnames(
-    "remove-tag",
-    "usa-button--unstyled",
-    hasAllFacilityAccess && "remove-tag--disabled"
-  );
+  const boxes = [
+    {
+      value: "ALL_FACILITIES",
+      label: `Access all facilities (${allFacilities.length})`,
+      disabled: isAdmin,
+    },
+    ...allFacilities.map((facility) => ({
+      value: facility.id,
+      label: facility.name,
+      disabled: isAdmin,
+    })),
+  ];
+  const checkedValues: { [key: string]: boolean | undefined } = {
+    ALL_FACILITIES: hasAllFacilityAccess,
+  };
+  allFacilities.forEach((facility) => {
+    checkedValues[facility.id] = userFacilityLookup.has(facility.id);
+  });
 
   return (
-    <React.Fragment>
-      <h3 className="margin-bottom-0">
-        Facility access{" "}
+    <>
+      <h3 className="testing-facility-access-subheader margin-bottom-0">
+        Testing facility access{" "}
         {showRequired && <span className="text-secondary-vivid">*</span>}
       </h3>
-      <p className="text-base">{facilityAccessDescription}</p>
+      <p className="testing-facility-access-subtext">
+        {facilityAccessDescription}
+      </p>
       <Checkboxes
```

Comment:
Would it be possible to check this formatting against a huge list of facilities, maybe 20-30+? We have some orgs with a lot of facilities, and want to make sure they can still see/access all the facilities with the new format: https://github.com/CDCgov/prime-simplereport/issues/3089

Reply:
Thank you!! Looks great. Love the names too 😄

Reply:
This looks fantastic, Nick!! +1 to the names. If the testing facility access check boxes are overwhelming, we can pivot the design where needed.